cloud/migrate-to-cloud/hypershift.md

```diff
@@ -218,6 +218,141 @@ To see all available configuration options for Hypershift, use this command:
 docker run -ti timescale/hypershift:0.4 --help
 ```
 
+### Filtering tables and schemas
+
+Hypershift offers filtering at the schema and table level. Filtering is
+enabled through the `include_schemas`, `exclude_schemas`, `include_tables`, and
+`exclude_tables` parameters in the hypershift configuration file. Some of these
+options are mutually exclusive.
+
+This is an example of a hypershift configuration file with schema and table
+exclusions:
+
+```yaml
+source: postgres://postgres:password@source:5432/database
+target: postgres://postgres:password@target:5432/database
+exclude_schemas:
+- public
+exclude_tables:
+- test.an_uninteresting_table
+```
+
+#### Pattern matching
+
+Hypershift does not support pattern matching on schema or table names in either
+inclusion or exclusion rules. Support may be provided for this in a future
+release.
+
+#### Include schemas
+
+The `include_schemas` configuration can be used to provide an array of schemas
+which should be included in the migration. Only objects in these schemas are
+copied.
+
+For example:
+
+```yaml
+include_schemas:
+- public
+- test
+- '"FooBar"'
+```
+
+
+<highlight type="note">
+It is possible for objects in one schema to depend on objects in another schema
+for example, a table in the `public` schema depending on a type in the `other`
+schema. Hypershift will not warn about missing dependencies on objects in
+schemas which are not included, instead the migration will fail.
+
+To avoid this, ensure that you do not exclude schemas containing objects on
+which objects in other schemas depend.
+</highlight>
+
+This does not work in combination with [exclude_schemas](#exclude-schemas).
+
+#### Exclude schemas
+
+The `exclude_schemas` configuration can be used to provide an array of schemas
+which should not be included in the migration. Only objects which are not in
+these schemas are copied.
+
+For example:
+
+```yaml
+exclude_schemas:
+- public
+- test
+- '"FooBar"'
+```
+
+<highlight type="note">
+It is possible for objects in one schema to depend on objects in another schema
+for example, a table in the `public` schema depending on a type in the `other`
+schema. Hypershift will not warn about missing dependencies on objects in
+excluded schemas, instead the migration will fail.
+
+To avoid this, ensure that you do not exclude schemas containing objects on
+which objects in other schemas depend.
+</highlight>
+
+This does not work in combination with [include_schemas](#include-schemas)
+
+#### Include tables
+
+The `include_tables` configuration can be used to provide an array of tables
+which should be included in the migration. Only these tables are copied.
+
+For example:
+
+```yaml
+include_tables:
+- '"MixedCaseSchema"."MixedCaseTable"'
+- public.my_table
+- public.a_small_table
+- other_schema.table
+```
+
+<highlight type="note">
```

Comment:
If my understanding is right, I remember, table-based migration will end up with the table in the target database in the `public` schema by default, even if it's part of another schema in the source database. If this is true, can we outline that here?